packages/client/src/utils.ts

```diff
@@ -68,3 +76,39 @@ export const createDefaultOnRedirect = () => {
     window.location.assign(url);
   };
 };
+
+export const generateCallbackUri = (uriInfo: UriInfo): string => {
```

Comment:
could use the destruct feature
`({ redirectUri, code, state, error, errorDescription }: UriInfo) => {} `